### DISCUSSION TITLE
是否有必要将`下载客户端`和`大会员`两个入口也做进`自定义顶栏`&功能适配更新提醒

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#5056] [功能建议 / Ideas]
开 #5055 顺便想到的（

是否有必要跟随原版顶栏将`下载客户端`和`大会员`两个入口也做进`自定义顶栏`：
![image](https://github.com/user-attachments/assets/7ae1819e-96f5-41a9-9491-96084e750aa0)
![image](https://github.com/user-attachments/assets/507a3ff4-e063-4d98-ac26-7cc8c22cf18a)

虽说这俩基本上就约等于破站自己给自己的广告位，就是怕有用户没了原版顶栏就找不到这俩入口（

***

顺便提醒新歌热榜的 URL 改成了：<https://music.bilibili.com/pc/music-center/> （没搞错的话这整个页面应该都改了）
![image](https://github.com/user-attachments/assets/f914a813-9712-416b-8ca4-b55a6f5b6e53)
`自定义顶栏`也没在这个页面生效。

**Comment by the1812** [comment]
这两个入口没有什么意义, 就不做了
